games/static/games/proto.js

```diff
@@ -163,12 +191,15 @@ class Board {
         this.setup = setup
         this.setupObject = setup.getAlienLocations()
```

Comment:
those 2 lines are confusing
whats the difference between setup and setupObject ?